packages/formstr-sdk/src/formstr/formstr.ts

```diff
@@ -712,3 +714,131 @@ export const syncFormsOnNostr = async (
   pool.publish(defaultRelays, nip51event);
   pool.close(defaultRelays);
 };
+
+export const getFormTemplateWithPassword = async (
+  formId: string,
+  formPassword: FormPassword,
+): Promise<V1FormSpec> => {
+  const pool = new SimplePool();
+  let formIdPubkey = formId;
+  let relayList = defaultRelays;
+  if (formId.startsWith("nprofile")) {
+    const { pubkey, relays } = nip19.decode(formId)
+      .data as nip19.ProfilePointer;
+    formIdPubkey = pubkey;
+    relayList = relays || defaultRelays;
+  }
+  const filter = {
+    kinds: [0],
```

Comment:
this functions will be updated to a new kind, how will backwards compatibility be maintained? two queries? or should it be maintained at the app level?